### DISCUSSION TITLE
Setting up for the first time

### DISCUSSION THREAD

**Discussion by diggerydoo68** [ChickenDevs/webcast#7] [Q&A]
Hello, great information here on this repo. Trying to set up the webcast for the first time. Never set up a webcast before in my life. Beforehand we were streaming with zoom with a webcam and laptop but was interested in testing this setup. I am able to log onto the config page. I feel like something is missing. So how exactly do users connect to the stream? Ultimately I would like to stream with zoom but the documentation states that you need give zoom the rtmp url and the stream key. Do we need to install that server on the pi or is it already installed? 

- thanks

**Comment by ChickenDevs** [comment]
Unfortunately you are not able to stream to zoom using this system, because zoom requires a desktop environment, and it would be completely separate automation.

You are able to broadcast to Facebook, twitch, YouTube, etc. We are currently just finishing up a fully automated YouTube system to replace the church locking us out of webcast.

For now, you are able to manually input the rtmp url and key on the config page, from any of the prior listed services. If you need any help setting this up, please let me know!

**Reply by diggerydoo68** [reply]
Good information to know. I'm still confused about a few things:

What service are you all currently streaming with? Is it Webcast? 

How are users connecting via Webcast? 

For our chapel it appears to be blocking YouTube live streaming. At least I get an error every time I try on YouTube live.  Is the network policies/firewall rules configured at ward level or stake level? 

Thanks for your help!

**Reply by ChickenDevs** [reply]
We were using webcast, but the church locked us out on general conference weekend, so we are currently using YouTube.

We just use a bit.ly link to our wards YouTube channel page for members to join. They can also search for {Ward Name} Sacrament Meeting.

The hard part of YouTube is you have to have an off-site server.

The content filters are managed by the stake technology specialist, and I believe it is per building.

**Reply by diggerydoo68** [reply]
Been a little bit, but I was able to set up the webcam and start streaming via YouTube Live. One thing that I now can't quite figure out is what is a good way to pause the stream? Not sure about you all but we are not allowed to stream the sacramental ordinance. Instead of starting and stopping the service. Is there a way to place an image over the stream, what encoders are you all using if any? To give more clarity, YouTube live times out after 2 mins when you stop the service from the cockpit config UI, how do I keep YouTube live from timing out?

**Reply by ChickenDevs** [reply]
Did you get the network switch and remote power switch? That's how we've been testing ours. I am also in the process of making an Android app, that'll be able to switch to an image instead of going to nothing.

Edit: Just saw your last question. We turn on auto start and turn off auto stop on YouTube.
